### PR TITLE
feat: Add javascript action example - view and edit timelogs

### PR DESCRIPTION
**Pull request by gismya**
This PR adds an example of registring and running actions using Node.js

The action in question is to view and edit timelogs for tasks.

### CODE REVIEW

File: javascript/actions/edit_timelogs/index.mjs [hunk 1]
```diff
@@ -0,0 +1,167 @@
+import { Session } from "@ftrack/api";
```
Comment (hdd) [inline javascript/actions/edit_timelogs/index.mjs:3]: Add copyright notes (eg:  # :copyright: Copyright (c) 2018 ftrack)

File: javascript/actions/edit_timelogs/index.mjs [hunk 2]
```diff
@@ -0,0 +1,167 @@
...
+const FTRACK_SERVER_URL = "";
+const FTRACK_API_USER = "";
+const FTRACK_API_KEY = "";
+class TimelogEditAction {
```
Comment (hdd) [inline javascript/actions/edit_timelogs/index.mjs:8]: Add space between class and const/s for better readability

**Review by hdd — COMMENTED** [review]
Might worth thinking of adding a top level readme on the Javascript level to explain how to run the code in general . You could simply move the # Installation part of this README up there.

**Comment by gismya** [comment]
I suggest we add general Javascript instructions as a separate task, to not stall this.

**Review by hdd — APPROVED** [review]

File: javascript/actions/edit_timelogs/index.mjs [hunk 3]
```diff
@@ -0,0 +1,170 @@
+// :copyright: Copyright (c) 2023 ftrack
+
+import { Session } from "@ftrack/api";
+const FTRACK_SERVER_URL = "";
```
Comment (lucaas) [inline javascript/actions/edit_timelogs/index.mjs:4]: Readme says these should be set as env vars - we should read them as such.

**Pull request merged**